content/blog/2021/03/camel-quarkus-release-1.8.0/index.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Camel Quarkus 1.8.0 Released"
+date: 2021-03-31
+authors: ["ppalaga"]
+categories: ["Releases", "Camel Quarkus"]
+preview: "Camel Quarkus 1.8.0 Released"
+summary: "The highlights of Camel Quarkus 1.8.0"
+---
+
+<sub><sup>Image by <a href="https://www.flickr.com/photos/lanzen/5984113332">Anders Lanzen</a> <a href="https://creativecommons.org/licenses/by-nc-sa/2.0">CC BY-NC-SA 2.0</a></sup></sub>
+
+We are pleased to announce the release 1.8.0 of Camel Quarkus! Here are the highlights.
+
+## Camel 3.9.0
+
+Camel 3.9.0 is rich in optimizations. It
+[reduces object allocations](https://camel.apache.org/blog/2021/03/Camel39-Whatsnew/#reduced-object-allocations),
+[optimizes Camel Core](https://camel.apache.org/blog/2021/03/Camel39-Whatsnew/#optimized-core) and
+the [HTTP component](https://camel.apache.org/blog/2021/03/Camel39-Whatsnew/#optimized-http-component).
```

Comment:
I think you'll need to use site relative links here. The reason being that the link checker doesn't check remote links, so we enforce all links that point to camel.apache.org to be site relative so they can be checked and we can notice broken links.

```suggestion
[reduces object allocations](/blog/2021/03/Camel39-Whatsnew/#reduced-object-allocations),
[optimizes Camel Core](/blog/2021/03/Camel39-Whatsnew/#optimized-core) and
the [HTTP component](/blog/2021/03/Camel39-Whatsnew/#optimized-http-component).
```